dev/scripts/generate-tarball.sh

```diff
@@ -61,36 +56,13 @@ function prepare_repo {
 
 function build {
   cd "${REPO_COPY}" > /dev/null
-  local build_all_client_profiles=$1
-  local hadoop_profile=$2
-  local client_profiles_=( )
-  if [[ "${build_all_client_profiles}" == true ]]; then
-    client_profiles=( ${EXTRA_ACTIVE_CLIENT_PROFILES[@]} )
-  fi
-  client_profiles+=( "default" )
-  mkdir -p "${CLIENT_DIR}"
-  for profile in "${client_profiles[@]}"; do
-    echo "Running build ${profile} and logging to ${BUILD_LOG}"
-    local profiles_arg="-Pmesos"
-    if [[ ${profile} != "default" ]]; then
-      profiles_arg+=" -P${profile}"
-    fi
-    if [[ ${hadoop_profile} != "default" ]]; then
-      profiles_arg+=" -P${hadoop_profile}"
-    fi
-    mvn -T 4C clean install -Dmaven.javadoc.skip=true -DskipTests -Dlicense.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true ${profiles_arg} ${BUILD_OPTS} > "${BUILD_LOG}" 2>&1
-    mkdir -p "${CLIENT_DIR}/${profile}"
-    local version=$(bin/alluxio version)
-    cp "core/client/runtime/target/alluxio-core-client-runtime-${version}-jar-with-dependencies.jar" "${CLIENT_DIR}/${profile}/alluxio-${version}-${profile}-client.jar"
-  done
-  if [[ "${build_all_client_profiles}" == true ]]; then
-    for profile in "${EXTRA_SYMLINK_CLIENT_PROFILES[@]}"; do
-      mkdir -p "${CLIENT_DIR}/${profile}"
-      pushd "${CLIENT_DIR}/${profile}" >> /dev/null
-      ln -s "../default/alluxio-${version}-default-client.jar" "alluxio-${version}-${profile}-client.jar"
-      popd >> /dev/null
-    done
+  local hadoop_profile=$1
+  echo "Running build and logging to ${BUILD_LOG}"
+  local profiles_arg="-Pmesos"
```

Comment:
Unrelated to this PR, should we remove the mesos profile and make it built by default?

Reply:
I am cool with this proposal, though I don't have too much context on the mesos integration. @aaudiber or @madanadit may have a better idea.

Reply:
+1 for removing the profile